README.rst

```diff
@@ -1,27 +1,29 @@
-****************************
+***************
 Mopidy-Webhooks
-****************************
+***************
 
 .. image:: https://img.shields.io/pypi/v/Mopidy-Webhooks.svg?style=flat
-    :target: https://pypi.python.org/pypi/Mopidy-Webhooks/
+    :target: https://pypi.org/project/Mopidy-Webhooks
     :alt: Latest PyPI version
 
 .. image:: https://img.shields.io/pypi/dm/Mopidy-Webhooks.svg?style=flat
-    :target: https://pypi.python.org/pypi/Mopidy-Webhooks/
+    :target: https://pypi.org/project/Mopidy-Webhooks
     :alt: Number of PyPI downloads
 
-.. image:: https://img.shields.io/travis/paddycarey/mopidy-webhooks/master.png?style=flat
+.. image:: https://github.com/paddycarey/mopidy-webhooks/workflows/CI/badge.svg?branch=master
     :target: https://travis-ci.org/paddycarey/mopidy-webhooks
-    :alt: Travis CI build status
+    :target: https://github.com/paddycarey/mopidy-webhooks/actions?query=branch%3Amaster+workflow%3ACI
+    :alt: Github Actions CI build status
 
 .. image:: https://img.shields.io/coveralls/paddycarey/mopidy-webhooks/master.svg?style=flat
    :target: https://coveralls.io/r/paddycarey/mopidy-webhooks?branch=master
    :alt: Test coverage
```

Comment:
This probably needs updating to point at codecov, but you'd need to sign up there to get the coverage data flowing.

Reply:
Let's remove codecov for now, and once the action is running post-merge we can add it back (just trying to minimise the number of new things being merge at once).